### ISSUE TITLE
make updategenerated generates code that fails vet check

### ISSUE THREAD

**Issue by davecheney** [heptio/contour#631]
Blocks #627

```
% make updategenerated
Updating CRD generated code...
Generating deepcopy funcs
Generating clientset for contour:v1beta1 at github.com/heptio/contour/internal/generated/clientset
Generating listers for contour:v1beta1 at github.com/heptio/contour/internal/generated/listers
Generating informers for contour:v1beta1 at github.com/heptio/contour/internal/generated/informers
```
generates code which fails the vet check
```
# github.com/heptio/contour/internal/generated/clientset/versioned/fake
internal/generated/clientset/versioned/fake/clientset_generated.go:56: literal copies lock value from fakePtr: github.com/heptio/contour/vendor/k8s.io/client-go/testing.Fake
```

**Comment by stevesloka** [comment]
After regenerating, I just roll back these two files:

```
 git checkout -- internal/generated/clientset/versioned/fake/register.go 
git checkout -- internal/generated/clientset/versioned/fake/clientset_generated.go 
```

**Comment by davecheney** [comment]
Thanks Steve. This isn't critical, and maybe will be solved upstream when they move up to Go 1.10. Bumping til 0.7

**Comment by stevesloka** [comment]
https://github.com/kubernetes/kubernetes/pull/60584
